packages/ui-kit/src/utils/side-by-side-content-styles.js

```diff
@@ -0,0 +1,21 @@
+import { css } from '@emotion/core';
+import { dimensions } from '../design-system';
+
+const sideBySideContainerStyle = css`
+  margin-top: 0;
+  > * {
+    margin-top: ${dimensions.spacings.m};
+  }
+  @media screen and (${dimensions.viewports.largeDesktop}) {
+    max-width: 100% !important;
+    display: grid;
+    grid-template-columns: 1fr ${dimensions.widths.pageContentSmall};
+    grid-template-rows: auto;
+    grid-column-gap: ${dimensions.spacings.xl};
+    grid-row-gap: ${dimensions.spacings.xl};
+    justify-items: stretch;
+    align-items: start;
+  }
+`;
+
+export default sideBySideContainerStyle;
```

Comment:
PS: this should be a styled component, no need to export only the styles.

```
const SideBySide = styled.div`
  ...
`
```

Reply:
> this should be a styled component, no need to export only the styles.

It's used in more than one component (here e.g. the API type and the manual `<SideBySide>` Block, plus more in the future) _and_ is sensitive to the block nesting levels (it needs to be the direct element under the MDX `<section>` wrappers to override the 770px constraint I liked it better this way, more like a mixin. 

I can try whether it works as a styled component without having to refactor too much though, maybe it was theoretical.

Reply:
In the current implementation you're passing the styles to a div element anyway, so from that point of view nothing changes if you export it as a component and replace the div elements.

Reply:
done, it's a component now